### DISCUSSION TITLE
Livewire V2, all data binding not working after upgrade frome V1

### DISCUSSION THREAD

**Discussion by nouart** [livewire/livewire#1576] [4. General]
### Description
After upgrading from V1 to V2 all data binding (select used for perpage, search) and pagination not working for the first time but after refresh manually  the page they works
Please how to solve the problem ?
### Steps to reproduce

**Context**
- Livewire version: [2.2.1]
- Laravel version: [7.28.1]
- Browser: [Chrome, v 87.0.4262.0 ]

**Comment by nuernbergerA** [comment]
Maybe `php artisan view:clear`?

Or is it on each page load?

**Reply by nouart** [reply]
I think is ` php artisan view:clear` 
Problem solved thanks
